src/cdn/polyfills.js

```diff
@@ -7,6 +7,7 @@ import 'core-js/stable/object/assign'
 import 'core-js/stable/object/entries'
 import 'core-js/stable/object/values'
 import 'core-js/stable/map'
+import 'core-js/stable/reflect'
```

Comment:
Just curious why we need the reflect polyfill

Reply:
wrap-promise.js uses `Reflect.construct` now

Reply:
Oh cool. Completely missed that.